### ISSUE TITLE
[BUG] Please add iOS version to package.swift

### ISSUE THREAD

**Issue by indieshack** [kylebrowning/APNSwift#106]
Wonderful project, thanks so much for this - totally saved my life! 

I'm not sure if the following is a bug or a feature request.

I'm working on a Xcode catalyst (multiplatform) project that uses APNSwift. The MacOS version compiles fine, but when trying to compile the project for iOS it gives the error: 

```The package product requires minimum platform version 13.0 for the iOS platform, but this target supports 9.0```

The reason for this is because APNSwift's Package.swift doesn't contain a definition for minimum iOS version - I fixed it for me by changing the APNSwift package definition to below, but since APNSSwift also works perfectly well on iOS should it have a version number for iOS as well as MacOS?

```let package = Package(
    name: "apnswift",
    platforms: [
        .macOS(.v10_15),
	.iOS(.v13),
    ], ...

**Comment by kylebrowning** [comment]
Sure thing!

**Comment by kylebrowning** [comment]
fixed in #122
